packages/api-server/scripts/extract_docs.py

```diff
@@ -34,7 +34,7 @@ def cleanup():
 outdir = f"{args.output}"
 os.makedirs(outdir, exist_ok=True)
 
-base_url = "http://localhost:8000/rmf-web"
+base_url = "http://localhost:8000"
```

Comment:
The docs is hosted at https://open-rmf.github.io/rmf-web/ and the config public url is `/rmf-web`, in theory this should point to `rmf-web` for it to work.

Reply:
Since we are not running the API server behind a proxy while collecting the docs, the `/rmf-web` public URL will not be applied, and everything needs to be scraped from `http://localhost:8000/docs` for example. However we will still need `/rmf-web` listed in the eventual html to work on ghpages

Reply:
For hosted environment, do you mean the docs hosting environment, https://open-rmf.github.io/rmf-web/?
We are populating the `/rmf-web` path using the docs retrieval API server config that still maintains the public URL `/rmf-web`

It is just the downloading of source files while it is hosted on localhost that needs to have no `/rmf-web` path